lib/pilot/tester_manager.rb

```diff
@@ -6,33 +6,26 @@ class TesterManager < Manager
     def add_tester(options)
       start(options)
 
-      begin
-        tester = Spaceship::Tunes::Tester::Internal.find(config[:email])
-        tester ||= Spaceship::Tunes::Tester::External.find(config[:email])
-
-        if tester
-          Helper.log.info "Existing tester #{tester.email}".green
-        else
-          tester = Spaceship::Tunes::Tester::External.create!(email: config[:email],
-                                                              first_name: config[:first_name],
-                                                              last_name: config[:last_name])
-          Helper.log.info "Successfully invited tester: #{tester.email}".green
-        end
+      app_id = find_app_id
 
-        app_filter = (config[:apple_id] || config[:app_identifier])
-        if app_filter
-          begin
-            app = Spaceship::Application.find(app_filter)
-            raise "Couldn't find app with '#{app_filter}'" unless app
-            tester.add_to_app!(app.apple_id)
-            Helper.log.info "Successfully added tester to app #{app_filter}".green
-          rescue => ex
-            Helper.log.error "Could not add #{tester.email} to app: #{ex}".red
-            raise ex
-          end
-        end
+      tester = Spaceship::Tunes::Tester::Internal.find_by_app(app_id, config[:email])
```

Comment:
These two lines could be extracted into one method. They are reused in a few places.